test/rest-api/params.mocks.ts

```diff
@@ -1,7 +1,17 @@
 import { setupWorker, rest } from 'msw'
 
+interface ResponseType {
+  username: string
+  messageId: string
+}
+
+interface RequestParams {
+  username: string
+  messageId: string
+}
+
 const worker = setupWorker(
-  rest.get(
+  rest.get<any, ResponseType, RequestParams>(
```

Comment:
I've added this usage example for the reference.

Reply:
This line gave me some ideas to fix **typescript** errors, thank you!